javaclient/pom.xml

```diff
@@ -105,9 +112,15 @@
         <artifactId>maven-assembly-plugin</artifactId>
         <version>2.5.5</version>
         <configuration>
-          <descriptorRefs>
-            <descriptorRef>jar-with-dependencies</descriptorRef>
-          </descriptorRefs>
+          <!-- not append assembly id in release file name -->
```

Comment:
Is there a reason why you switched from jar-with-dependencies to separate assembly?

Reply:
i used to package all the dependency into lib,but it's ok to use jar-with-dependencies